playwright/pages/SliceMachinePage.ts

```diff
@@ -2,11 +2,16 @@ import { Locator, Page } from "@playwright/test";
 
 import { Menu } from "./components/Menu";
 import { InAppGuideDialog } from "./components/InAppGuideDialog";
+import { EnvironmentSelector } from "./components/EnvironmentSelector";
+import { LoginDialog } from "./components/LoginDialog";
 
 export class SliceMachinePage {
   readonly page: Page;
+  readonly topBorder: Locator;
   readonly menu: Menu;
   readonly inAppGuideDialog: InAppGuideDialog;
+  readonly environmentSelector: EnvironmentSelector;
```

Comment:
⚠️ Environment selector should be in Menu for me, it's today specific of this area. WDYT?

Reply:
Works for me! Good catch.